docs/recipes/Enabling-Serverside-Rendering.md

```diff
@@ -2,8 +2,103 @@
 [👈 Return to Overview](./README.md)
 <!--ONE-DOCS-HIDE end-->
 
-# Enabling Serverside Rendering
+# Enabling Server-side Rendering
 
-Recipe is forthcoming.
+One App will always attempt render your module on the server before sending html back to the client. However, it won't wait for any asynchronous tasks, such as data fetching, to complete before doing so.
+
+The following options provide ways to ensure that server side requested data is included as part of the initial render.
+
+## Using `loadModuleData`
+
+One App v5 introduced [`loadModuleData`](https://one-amex-docs.americanexpress.com/en-us/one-app/api/modules/loading-data) which is invoked on the server before rendering a module.
+
+```jsx
+YourModule.holocron = {
+  name: 'your-module',
+  // Runs on both Server and Browser
+  loadModuleData: async ({ store, fetchClient, ownProps }) => {
+    store.dispatch({ type: 'LOADING_API' });
+    const response = await fetchClient('https://api.example.com', ownProps.options);
+    const data = await response.json();
+    store.dispatch({ type: 'LOADED_API', data });
+  },
+};
+```
+
+## Using Fetchye and `loadModuleData`
+
+```jsx
+import React, { Fragment } from 'react';
+import ImmutableCache from 'fetchye-immutable-cache';
+import { useFetchye, makServerFetchye } from 'fetchye';
+
+const BookList = () => {
+  const { isLoading, data } = useFetchye('http://example.com/api/books/');
+
+  if (isLoading) {
+    return (<p>Loading...</p>);
+  }
+
+  return (
+    <Fragment>
+      {/* Render data */}
+    </Fragment>
+  );
+};
+
+BookList.holocron = {
+  loadModuleData: async ({ store: { dispatch, getState }, fetchClient }) => {
+    if (global.BROWSER) {
```

Comment:
Did we find out which one includes / removes the code from the client bundle? `global.BROWSER` or `!global.BROWSER`

Reply:
So `if (!global.BROWSER)` would prevent it from being bundled in the client code